Source/Core/DotSpatial.Projections/DotSpatial.Projections.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
why did you remove net6.0?

Reply:
The reason is that multitargeting for this project is not required. NetStandard library will run with both .NET 6 Application and other netstandard libraries.